src/routes/accommodationRoute.js

```diff
@@ -81,16 +82,18 @@ const fUpload = fileUpload({
 });
 
 const {
-  isSupplierAccommodation, checkForImages, checkForImagesUpdate
+  isSupplierAccommodation, checkForImages, checkForImagesUpdate,
 } = accMidd;
 const { checkToken } = userMidd;
 const {
-  addAccommodation, getAccommodation, deleteAccommodation, editAccommodation
-} = accommodationCont;
+  addAccommodation, getAccommodation, deleteAccommodation, editAccommodation,
+} = accommodationController;
 
 app.patch('/:id', fUpload, checkToken, roles.checkHost, isSupplierAccommodation, checkForImagesUpdate, valid.editAccommodation, editAccommodation);
 app.delete('/:id', checkToken, roles.checkHost, isSupplierAccommodation, deleteAccommodation);
 app.post('/', fUpload, checkToken, roles.checkHost, valid.accommodation, checkForImages, addAccommodation);
 app.get('/', checkToken, accMidd.checkViewAccommodation, getAccommodation);
+app.get('/:id/rating', valid.getReviewvalidation, bookingsController.getAverageRating);
```

Comment:
No need for two routes. We only need one route to handle both average ratings and feedback/comments.
`/:accommodationId/ratings` -> e.g `{averageRating: 3.45, feedbackList: ['string', 'strings']}`